src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -5148,10 +5155,11 @@ def create_public_ip(cmd, resource_group_name, public_ip_address_name, location=
         tier = pip_obj['sku']['tier']
         zone = pip_obj['zones'] if 'zones' in pip_obj else None
 
-    if sku is None:
+    if sku is None or sku.lower() == "basic":
```

Comment:
As most of the warning messages are duplicated, it's better to extract the warning message as shared text as follows.

`It's recommended to create with {0}. Please be aware that the default {1} will be changed from Basic to Standard in the future and Basic option will be removed.`

Reply:
Can we update the warning message to read as the following:

"It's recommended to create with {0}. Please be aware that the default {1} will be changed from Basic to Standard in the next release. Also note that Basic option will be removed in the future."